app/actors/hyrax/actors/collections_membership_actor.rb

```diff
@@ -12,28 +12,29 @@ class CollectionsMembershipActor < AbstractActor
       # @param [Hyrax::Actors::Environment] env
       # @return [Boolean] true if create was successful
       def create(env)
-        attributes_collection = env.attributes.delete(:member_of_collections_attributes)
-        assign_nested_attributes_for_collection(env, attributes_collection) &&
-          next_actor.create(env)
+        assign_nested_attributes_for_collection(env) && next_actor.create(env)
```

Comment:
Moving the assignment of attributes_collection to be inside `assign_nested_attributes_for_collection(env) will impact collections-sprint which has this code...
```
def create(env)
        attributes_collection = env.attributes.delete(:member_of_collections_attributes)
        extract_collection_id(env, attributes_collection)
        assign_nested_attributes_for_collection(env, attributes_collection) &&
          next_actor.create(env)
end
```

Reply:
Note that `extract_collection_id(env, attributes_collection)` is not part of #update.